src/app/logic/correctAnswer.js

```diff
@@ -0,0 +1,7 @@
+const isAnswerCorrect = (correctAnswer, chosenAnswer) => {
+  return correctAnswer === chosenAnswer;
+};
+
```

Comment:
If you want to use this function outside of this file, you will need to add export to it.

Reply:
Oh god, I forgot about it...